components/user/ModalCollection.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <!-- TODO: move texts to translations -->
+  <b-modal
+    id="modal-collection"
+    hide-header
+    hide-footer
+  >
+    <div v-if="showForm">
+      <h2>Create new collection</h2>
+      <b-form @submit.stop.prevent="submitForm">
+        <b-form-group
+          label="Collection name"
+          label-for="collection"
+        >
+          <b-form-input
+            id="collection"
+            type="text"
+            required
+          />
```

Comment:
Shouldn't the form input demand the use of a validation method, to ensure the ticket's acceptance criteria (The title should have a maximum length of [whatever our card title max-length is])? As well as the one about the description input (The description should have a maximum length of 240 characters.).

Reply:
Yes, thanks!